pagopa/io-app
# chore: [IOBP-521] Add transaction status polling before PSP selection screen

Depends on https://github.com/pagopa/io-app/pull/5439

## Short description
This PR adds the polling for transaction activation in the payment flow

## List of changes proposed in this pull request
- Add `useOnTransactionActivationEffect` custom hook which handles the polling for the transaction status
- Added status polling before navigate to the PSP selection screen

## How to test
With the `io-dev-api-server`, checkout [this branch](https://github.com/pagopa/io-dev-api-server/pull/342), from **Profile > Playground > New Wallet > Payment** start a payment flow and check that the payment can be finalized.
